Expensify/App
# Added frequently used emojis support

### Details

- Added support for Frequently Used Emojis and Recently used emojis
- Belongs to a common list and it is synced across sessions with NameValuePairs

### Fixed Issues

$ https://github.com/Expensify/App/issues/4559

### Tests
1. Tested adding multiple emojis to recent
2. Tested emojis by multiple counts (frequency)
3. Tested by limiting the emoji count and relevant replacement logic in case of more emojis than the given limit
4. Tested sync across sessions for Frequent Emoji List
5. Tested preferred skin tone along with frequent emojis


### QA Steps
1. Go to any chat and open EmojPicker
2. Select your favorite emoji and add it to Text
3. Open the EmojiPicker again and see it as a part of the Frequently Used Emojis
4. Add multiple emojis to see that the order is either based on the frequency or recently added emojis



### Tested On


### Screenshots


#### Web

<img width="388" alt="web-frequent-emojis" src="https://user-images.githubusercontent.com/3069065/143644825-483717a8-0741-48b3-95a6-371a8a80d7bf.png">


#### Mobile Web

<img width="395" alt="mweb-frequent-emojis" src="https://user-images.githubusercontent.com/3069065/143644944-ff3c90e7-b8b9-41aa-b6ef-558c8eaa04bf.png">

#### Desktop

<img width="431" alt="desktop-frequent-emojis" src="https://user-images.githubusercontent.com/3069065/143645173-6ca537a1-35ac-45a9-8610-dd816a7383a5.png">

#### iOS

<img width="407" alt="ios-frequent-emojis" src="https://user-images.githubusercontent.com/3069065/143645274-f392f96f-99cc-41f8-97ce-872e85615f48.png">

#### Android

<img width="349" alt="android-frequent-emojis" src="https://user-images.githubusercontent.com/3069065/143645405-847b48e6-4b92-427e-a700-f74900c3deea.png">
